test/spec-cpp.sh

```diff
@@ -187,15 +193,24 @@ summary-tsv-row() {
   local spec_subdir=$1
   shift
 
+  local status cpp_failures_allowed cpp_failures
+
   if test $# -eq 1; then
     local spec_name=$1
     local -a tsv_files=( _tmp/spec/$spec_subdir/$spec_name.result.tsv )
+    status=`awk '{printf $1}' "_tmp/spec/$spec_subdir/$spec_name.task.txt"`
+    cpp_failures_allowed=`awk '{printf $6}' "_tmp/spec/$spec_subdir/$spec_name.stats.txt"`
+    cpp_failures=`awk '{printf $7}' "_tmp/spec/$spec_subdir/$spec_name.stats.txt"`
```

Comment:
Please use $() for command sub everywhere rather than backticks

(this could be a good rule for a linter)